pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/SymbolStack.java

```diff
@@ -45,6 +45,17 @@
 
 public class SymbolStack {
 
+    public static final List<String> noParenBuiltInFunc =
+            Arrays.asList(
+                    "SYS_DATE",
+                    "SYS_DATETIME",
+                    "SYS_TIME",
+                    "SYS_TIMESTAMP",
+                    "SYSDATE",
+                    "SYSDATETIME",
+                    "SYSTIME",
+                    "SYSTIMESTAMP");
+
```

Comment:
missed the below functions?

CURRENT_DATE
CURRENT_TIME
CURRENT_DATETIME
CURRENT_TIMESTAMP
LOCALTIME
LOCALTIMESTAMP

Reply:
Unlike the added 8 functions, current_date, etc do not cause the problem. That is, 
```
select current_date() from dual;
```
does not cause an error, while 
```
select sysdate() from dual;
```
does.